src/support.html

```diff
@@ -59,6 +59,27 @@
         Keep the two buttons pressed for at least 15 seconds. Now, the system
         should boot from the USB flash drive again.
       </p>
+  - title: >
+      Is Zigbee2MQTT supported by the on-board radio?
+    content: >
+      <p>
+        The Home Assistant Yellow team officially supports ZHA (Zigbee Home
+        Automation), the Zigbee stack integrated into Home Assistant Core.
+      </p>
+
+      <p>
+        The Zigbee2MQTT project has <a href="https://www.zigbee2mqtt.io/guide/adapters/#experimental">
+        experimental support</a> (in beta stage) for the Silicon Labs radio on
+        Home Assistant Yellow. With this setting Zigbee2MQTT is known to work
+        on Home Assistant Yellow.
+      </p>
+
+      <p>
+        Note: The Zigbee Home Automation integration is active by default! If
+        you are using Zigbee2MQTT, remember to delete and ignore the discovered
+        Zigbee Home Automation integration to avoid interference.
+      </p>
```

Comment:
```suggestion
      </p>
      
      <p>
        Note: When setting up Zigbee2MQTT with the Home Assistant Yellow, you will
        need to configure the adapter type in Zigbee2MQTT. Make sure the following
        is present in your Zigbee2MQTT configuration:
        
        <code>
          serial:
            adapter: ezsp
            port: dev/ttyAMA1
        </code>
      </p>

```

Reply:
Looks like the current CSS also needs preformatted tags to make it look nice

```suggestion
      </p>
      
      <p>
        Note: When setting up Zigbee2MQTT with the Home Assistant Yellow, you will
        need to configure the adapter type in Zigbee2MQTT. Make sure the following
        is present in your Zigbee2MQTT configuration:
        <pre><code>serial:
          adapter: ezsp
          port: /dev/ttyAMA1</code></pre>
      </p>
```